gef.py

```diff
@@ -2867,14 +2865,32 @@ def get_terminal_size():
     """Return the current terminal size."""
     if is_debug():
         return 600, 100
+    
+    if platform.system() == "Windows":
+        from ctypes import windll, create_string_buffer
+        hStdErr = -12 
+        herr = windll.kernel32.GetStdHandle(hSdErr)
```

Comment:
You didn't test your last change: this is missing a 't'

Reply:
Oh Sorry about that.
I fix this type and test on my windows. I think it should work without error this time
![image](https://user-images.githubusercontent.com/12622489/64073614-17bdef00-ccd3-11e9-86a7-f2abdc6372ee.png)